bootstrap.sh

```diff
@@ -6,8 +6,7 @@ GO111MODULE=on GOBIN="$(pwd)/util" go get -u github.com/Harvey-OS/ninep/cmd/ufs
 # GO111MODULE=on GOBIN="$(pwd)/util" go get -f -u bldy.build/bldy
 GO111MODULE=on GOBIN="$(pwd)/util" go get ./util/src/harvey/cmd/...
 
-GOOS=plan9 GOARCH=amd64 GO111MODULE=on GOBIN="$(pwd)/util" go get golang.org/dl/gotip
-
+GOOS=plan9 GOARCH=amd64 GO111MODULE=on go get golang.org/dl/gotip
```

Comment:
```suggestion
```

Reply:
Oh I see, thanks! I will wrap my head around that. :)

Reply:
Beside the home directory issue, you'd need to a working `git` command to clone the source and an existing Go distribution already installed to bootstrap Go. Maybe we can write a rc script to bootstrap go tip using the plan9 binaries on this page: http://9legacy.org/download.html

Reply:
Hey @orangecms I would like to commit this suggestion and merge this... is that ok?